### ISSUE TITLE
Does X-Ray really usefull for diagnosing COVID-19?

### ISSUE THREAD

**Issue by Goodsea** [ieee8023/covid-chestxray-dataset#5]
"March 4, 2020 -- X-ray may not be the best imaging tool for detecting novel coronavirus disease (COVID-19). Almost three-quarters of a small cohort of South Korean patients with COVID-19 pneumonia had normal chest x-rays, missing pulmonary nodules that chest CT identified, according to a February 26 study in the Korean Journal of Radiology."

https://www.auntminnie.com/index.aspx?sec=sup&sub=xra&pag=dis&ItemID=128347

**Comment by ieee8023** [comment]
This work supports it:
https://pubs.rsna.org/doi/10.1148/radiol.2020200432 and https://pubs.rsna.org/doi/10.1148/radiol.2020200642

And this recommends not using it: https://www.acr.org/Advocacy-and-Economics/ACR-Position-Statements/Recommendations-for-Chest-Radiography-and-CT-for-Suspected-COVID19-Infection

But I believe it could help. Imagine if we run out of tests and then the majority of radiologists get sick. I think AI tools can help general practitioners to treat patients.

**Comment by Goodsea** [comment]
I think science generally trying to improve itself by time. We can not say last research %100 true and first research %100 false absolutely, it depends. But I observe that some of the recent researchs about this topic showing that the chest-xrays doesn't help in high accuracy to diagnose this disease.

| Research | Publication Date | Help to Diagnose COVID-19 with Chest X-Ray |
|---|---|---|
| https://pubs.rsna.org/doi/10.1148/radiol.2020200432 | 19 Feb 2020 | Supports X-Ray
| https://pubs.rsna.org/doi/10.1148/radiol.2020200642 | 26 Feb 2020 | Supports X-Ray
| https://www.auntminnie.com/index.aspx?sec=sup&sub=xra&pag=dis&ItemID=128347 | 26 Feb 2020 | Doesn't Support X-Ray
| https://www.acr.org/Advocacy-and-Economics/ACR-Position-Statements/Recommendations-for-Chest-Radiography-and-CT-for-Suspected-COVID19-Infection | 11 March 2020 | Doesn't Support X-Ray

This research ("https://www.auntminnie.com/index.aspx?sec=sup&sub=xra&pag=dis&ItemID=128347") already saying that in some cases, there is correlation between Chest X-Ray and Diagnose COVID-19, but in some cases there isn't. 

As those trying to find the way to help diagnose and treat this disease, I think we should search other ways to help. At least we don't have enough large "COVID-19 Chest X-Ray Dataset" to train and test our machine learning models. 

BTW, this pandemic spreads quickly. So time is so important. I think we need to take the right steps at this time.

**Comment by SandeepAswathnarayana** [comment]
@ieee8023 and @Goodsea, I had the same question in mind, but with relevance to Chest CT Scans. After having read a few relevant research papers over the weeks, I just couldn't wrap my head around and come to a conclusion. I've always wanted to reach out to seasoned radiologists, meet with them in person and discuss some noteworthy insights. Attaining 'domain' knowledge has been a real challenge for me as I haven't worked exclusively in Healthcare (say, Radiology in this case).

I'd recommend this video [Diagnosing COVID-19 with Chest CT Findings](https://www.youtube.com/watch?v=7de8LJE4owg&t=268s) in which the speaker talks about the 3 major CT findings (Ground-glass opacities, Consolidations, and Crazy paving patterns) as the signs of COVID-19 detection. He also mentions its limitations and some key CT findings that are typically absent in COVID-19.

**Comment by rcillavicomtech** [comment]
I've been wondering about this too. As a research engineer I think that our mission right now is to create the best possible models to help in the diagnosis and prognosis of COVID-19 patients. Then physicians would be the ones that decide which are the right tools to employ.

Evidence shows that RT-PCA is the best tool right know for COVID-19 diagnosis and I don't think imaging is going to replace it. But still we need to do some effort on improving diagnosis models to employ them as a backup or in remote areas where diagnosis kits are not available. 

I think patient prognosis is where there is more room for intelligent image analysis methods. An imaging biomarker to quantify how pneumonia has affected the lungs of the patient could be an useful tool for physicians to track their evolution and eventually to help in therapy prescription.

**Comment by maxwelljohn** [comment]
This paper got an accuracy rate of 95% using deep learning on CT scans:

https://www.medrxiv.org/content/medrxiv/early/2020/02/26/2020.02.25.20021568.full.pdf

This paper, which also used deep learning on CT scans, had an accuracy of only 87%.  However, it looks like they are specifically focused on early-stage screening.  In the abstract, they claim that RT-PCR doesn't work very well in the early stage:

https://arxiv.org/pdf/2002.09334.pdf

I found both these papers by doing keyword searches on Google Scholar, btw.  I haven't taken a careful look at either.

**Comment by AdarshPan** [comment]
A lot of the papers and Websites are saying that Chest CT is the most preffered way of detecting the virus........as data scientists we have to do whatever possible to help the community.....does anyone knows about the site where wen can get a Chest CT of The COVID

**Comment by Goodsea** [comment]
> A lot of the papers and Websites are saying that Chest CT is the most preffered way of detecting the virus........as data scientists we have to do whatever possible to help the community.....does anyone knows about the site where wen can get a Chest CT of The COVID

We should take a step for the pandemic, you are definitely right.

**Comment by SandeepAswathnarayana** [comment]
UPDATE:
FDA grants emergency use authorization for the fastest available (as of March 28) molecular point-of-care test for COVID-19: [Abbott Laboratories](https://abbott.mediaroom.com/2020-03-27-Abbott-Launches-Molecular-Point-of-Care-Test-to-Detect-Novel-Coronavirus-in-as-Little-as-Five-Minutes)

**Comment by atbenmurray** [comment]
I guess there are a few aspects to 'why xray over chest ct'. We got briefed on this a few days back when we started looking into building deep learning pipelines, and I took the following conclusions from it:
1. **Throughput**: You can x-ray a patient with a very rapid turnaround and little or no decontamination between patients. CT would need a decontamination between every patient. One figure we got from clinicians (indirectly) was about two to three patients an hour, max
2. **Radiation exposure**: Ideally you might want to obtain data from a patient as the disease progresses. This might represent an unacceptable level of radiation
3. **Size of dataset**: Xraying a large volume of patients and a low dosage of radiation permits a much larger dataset collection, along with the ability to have longitudinal data. A sufficiently large number of xrays, particularly with multiple time points per patient, may allow a network to overcome the lack of information present in the xray vs. a ct
I hope this helps.

**Comment by ieee8023** [comment]
I think focusing on making prognostic instead of diagnostic predictions makes more sense in terms of the need now.

**Comment by SandeepAswathnarayana** [comment]
> I think focusing on making prognostic instead of diagnostic predictions makes more sense in terms of the need now.

@ieee8023 You're right. This is where the domain expertise, clinical trials in fields like Genetics, Virology, etc. add substantial value.

While we are aware of why the [antibody tests matter](https://www.sciencenews.org/article/covid-19-coronavirus-pandemic-how-antibody-blood-tests-work), it's also important to address the 'value' researchers are adding to help [eliminate theories](https://www.sciencenews.org/article/coronavirus-covid-19-not-human-made-lab-genetic-analysis-nature) - which at times could otherwise be taken at face value.

P.S. This repository still does justice to the intent and motivation with which it was started. It's only fair to compare one's vision to his/her motivation. Cheers!

**Comment by ncovgt2020** [comment]
@atbenmurray do you happen to have any references for the  throughput, radiation exposure and size of dataset arguments?
